projects/habitat_objectnav/configs/agent/hm3d_eval.yaml

```diff
@@ -44,6 +44,8 @@ AGENT:
     dilate_obstacles: False
     dilate_size: 3
     dilate_iter: 1
+    record_instance_ids: True
```

Comment:
What happens if these new fields aren't filled out? Do we need them in the real robot configs?

Reply:
added default values for now and tested